testnet/testnet-deploy-l2-contracts.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+#
+# This script deploys contracts to testnet
+#
+
+help_and_exit() {
+    echo ""
+    echo "Usage: $(basename "${0}") --l1host=gethnetwork --pkstring=f52e5418e349dccdda29b6ac8b0abe6576bb7713886aa85abea6181ba731f9bb"
+    echo ""
+    echo "  l2host             *Required* Set the l2 host address"
+    echo ""
+    echo "  obxpkstring           *Required* Set the pkstring to deploy OBX contract"
+    echo ""
+    echo "  ethpkstring           *Required* Set the pkstring to deploy ETH contract"
+    echo ""
+    echo "  l2port             *Optional* Set the l2 port. Defaults to 10000"
+    echo ""
+    echo ""
+    echo ""
+    exit 1  # Exit with error explicitly
+}
+# Ensure any fail is loud and explicit
+set -euo pipefail
+
+# Define local usage vars
+start_path="$(cd "$(dirname "${0}")" && pwd)"
+testnet_path="${start_path}"
+
+# Define defaults
+l2port=13000
+obxpkstring="6e384a07a01263518a09a5424c7b6bbfc3604ba7d93f47e3a455cbdd7f9f0682"
+ethpkstring="4bfe14725e685901c062ccd4e220c61cf9c189897b6c78bd18d7f51291b2b8f8"
```

Comment:
Why 2 different pk string ?

Reply:
Tipicaly required fields don't have a default

Reply:
Yeah sorry, 'required' description was out of date. I think we should default them for now, can make them secret as another piece of work (I'll add todo/raise ticket)